docs/Installing-Lmod-without-root-permissions.rst

```diff
@@ -51,11 +51,11 @@ Lmod
 ~~~~
 
 **Step 1**: Download and unpack the latest available Lmod version,
-`Lmod-6.0.1.tar.bz2`_ at the time of writing.
+`Lmod-6.1.tar.bz2`_ at the time of writing.
```

Comment:
I'm sticking to the releases on SourceForge here (although, I must admit, I'm not sure what algorithm @rtmclay uses for pushing stuff to SourceForge or not...)